src/System.Globalization/tests/CharUnicodeInfo/CharUnicodeInfoTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class CharUnicodeInfoGetUnicodeCategoryTests
+    {
+        [ActiveIssue(7351, PlatformID.AnyUnix)]
+        [Fact]
+        public void GetUnicodeCategory()
```

Comment:
It's not two code points that are failing. Others are too, but aren't being reported as one test failure stops all other tests